src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -1563,9 +1582,17 @@ ValidatorList::getJson() const
         curr[jss::available] =
             pubCollection.status == PublisherStatus::available;
 
-        auto appendList = [](PublisherList const& publisherList,
-                             Json::Value& target) {
-            target[jss::uri] = publisherList.siteUri;
+        auto appendList = [&context](
+                              PublisherList const& publisherList,
+                              Json::Value& target) {
+            if ((context && context->apiVersion == 1) || !context)
```

Comment:
This can be written as `if (!context || context->apiVersion == 1)`. Or if you get rid of the context object as I suggested above, it would be `if (!apiVersion || *apiVersion == 1)`.